src/main/java/book/store/onlinebookstore/exception/GlobalExceptionHandler.java

```diff
@@ -48,6 +49,11 @@ public ResponseEntity<String> handleAccessDenied(AccessDeniedException ex) {
         return new ResponseEntity<>(ex.getMessage(), HttpStatus.FORBIDDEN);
     }
 
+    @ExceptionHandler(JwtException.class)
+    public ResponseEntity<String> handleJwtException(JwtException ex) {
+        return new ResponseEntity<>(ex.getMessage(), HttpStatus.FORBIDDEN);

```

Comment:
well it depends on what stays behind `JwtException`

if you failed to parse and validate JWT - then HTTP 401 (UNAUTHORIZED) should be returned

if parsing and validation succeeded, but JWT is missing some grants - then HTTP 403 (FORBIDDEN) should be returned